src/simplewallet/simplewallet.cpp

```diff
@@ -10702,7 +10702,9 @@ int main(int argc, char* argv[])
     tools::signal_handler::install([&w](int type) {
       if (tools::password_container::is_prompting.load())
       {
-        // must be prompting for password so return and let the signal stop prompt
+        w.stop();
```

Comment:
Can you add a comment explaining the rationale for calling `deinit` and `_exit` here in addition to stop (in particular that it wasn't a UX design choice so much as a choice of necessity)? It doesn't have to be nearly as detailed as the PR description, but would probably help anyone trying to refactor in the input handling to allow a more graceful exit.